docs/source/support/must-gather.md

```diff
@@ -0,0 +1,63 @@
+# Gathering data with must-gather
+
+`must-gather` is an embedded tool in Scylla Operator that helps collecting all the necessary info when something goes wrong.
+
+The tool talks to the Kubernetes API, retrieves a predefined set of resources and saves them into a folder in your current directory.
+By default, all collected Secrets are censored to avoid sending sensitive data inside the archive.
+That said, you can always review the archive before you attach it to an issue or your support request.
+
+Given it needs to talk to the Kubernetes API, at very least, you need to supply the `--kubeconfig` flag with a path to the kubeconfig file for your Kubernetes cluster, or set the `KUBECONFIG` environment variable.
+
+## Running must-gather
+
+There is more than one way to run `must-gather`.
+Here are some examples of how you can run the tool.
+
+All examples assume you have exported `KUBECONFIG` environment variable that points to a kubeconfig file on your machine.
+If not, you can run this command to export the common default location.
+Please make sure such file exists.
```

Comment:
```suggestion
Please make sure such a file exists.
```

Reply:
i think with such you don't use `a`. say the common error message is also `No such file or directory`

Reply:
It's a bit more intricate; generally a singular noun requires an article. Meanwhile with "no", it'd be ungrammatical. The reason is that "no", as well as being a quantifier, is also a determiner, and cannot be accompanied by another determiner.

Reply:
updated, thx